Sources/GRPC/ClientConnection.swift

```diff
@@ -371,6 +371,11 @@ extension ClientConnection {
         return host
       }
     }
+    
+    if let hostname = serverHostname, hostname.isIPAddress {
+      logger.debug("IP address cannot be used for TLS SNI extension. No host used", metadata: ["server-hostname": "nil"])
```

Comment:
I wonder if the metadata should contain the `hostname` value that has been rejected?

Reply:
That's probably worth adding @esen

Reply:
done

Reply:
Thanks! Could you please use just `hostname` instead of `"\(hostname)"`?

Reply:
A bare string cannot be used here. `"\(hostname)"` is the [preferred](https://github.com/apple/swift-log/blob/master/Sources/Logging/Logging.swift#L289) way

Reply:
Oh, sorry!